package.json

```diff
@@ -33,7 +33,7 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
```

Comment:
@vitorbal @ngryman I just thought of this, but with the large leap in versions of `preset-env`, we may be able to remove these explicit dependencies for specific plugins in favor of just using whatever `preset-env` depends on.

Reply:
@msholty-fd makes sense! Do you know if `plugin-transform-classes` is also supported by `preset-env`?

Reply:
@vitorbal I do not! We can always check when we try it out in `zapier/zapier`